src/components/NewFooter/Footer.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { StyledFooter } from './Menu.style';
+import MenuContent from './MenuContent';
+
+const Footer: React.FC = () => {
+  return (
+    <StyledFooter>
```

Comment:
We need to add the attribute `role="contentinfo"` to the footer element to communicate the role for browsers that don't support the `footer` element